roles/monitoring_server/tasks/install_nginx_container.yml

```diff
@@ -1,14 +1,14 @@
 ---
 - name: Ensure passlib is installed on centos7
-  ansible.builtin.dnf:
+  ansible.builtin.yum:
```

Comment:
I've reverted this change from #109 which seemed to be causing some issues. But current bug is related to #111.